sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/FormRecognizerAsyncClient.java

```diff
@@ -133,9 +133,9 @@
      * @param length The exact length of the data. Size of the file must be less than 50 MB.
```

Comment:
do you validate the size in the client?
Also, is this constrain from the service?

Reply:
We do not validate the size on the client and yes it is a constraint from the service.

Reply:
I would prefer deleting this from here then, in case the service decides to change the limitations